src/app/examples/chip-example/chip-example.component.tns.html

```diff
@@ -0,0 +1,4 @@
+<WrapLayout orientation="horizontal" class="chip-example">
+  <kirby-chip label="kirby"></kirby-chip>
+  <kirby-chip label="kirby" isSelected="true"></kirby-chip>
```

Comment:
Rename to `text` as we use for buttons, badge etc.
```suggestion
  <kirby-chip text="kirby" isSelected="true"></kirby-chip>
```